### PR TITLE
Reset case name

### PR DESCRIPTION
**Pull request by mkangia**
https://dimagi-dev.atlassian.net/browse/ICDS-786

Added management command to serve a custom ICDS request to set names on cases to last entered value in case they are blank now.
This has happened due to a bug in the application which resulted in case names to be overwritten with empty value, for ex [this](https://www.icds-cas.gov.in/a/icds-cas/reports/case_data/7d9db454-1f7b-4022-824d-ca3277a817e1/) case.

### CODE REVIEW

**Comment by mkangia** [comment]
I have tested separate bits of this script by running them manually, i would eventually run this once without actually doing case updates and then do it with a small number of cases and then for all data.
As of now the number of such cases is ~100,000 for ICDS

**Review by snopoke — APPROVED** [review]

**Comment by mkangia** [comment]
Added the "do not merge" flag to test the change i made in [last commit](https://github.com/dimagi/commcare-hq/pull/25624/commits/3ffda0130a73aa0d8a8300ce4acfe7ede7d9b129).

File: corehq/apps/hqcase/utils.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def bulk_update_cases(domain, case_changes, device_id):
         # Ensure the XML is formatted properly
         # An exception is raised if not
         case_block = ElementTree.tostring(case_block.as_xml())
-        case_blocks.append(case_block)
+        case_blocks.append(case_block.as_text())
```
Comment (mkangia) [inline corehq/apps/hqcase/utils.py]: @snopoke 
this fails with
```
ipdb> case_block
b'<case case_id="122" date_modified="2019-10-18T15:51:10.465704Z" user_id="system" xmlns="http://commcarehq.org/case/transaction/v2"><update><name>motu</name></update></case>'
ipdb> case_block.as_text()
*** AttributeError: 'bytes' object has no attribute 'as_text'
ipdb> case_block.as_xml
```
works if i remove the `ElementTree` call and just do `case_blocks.append(case_block.as_text())` but then the comment above says `Ensure the XML is formatted properly` which would be skipped if we do just `as_text`.
should we reconsider using 
`case_block = ElementTree.tostring(case_block.as_xml()).decode('utf-8')`?

Comment (snopoke) [reply corehq/apps/hqcase/utils.py]: The intention was this:
```diff
- case_block = ElementTree.tostring(case_block.as_xml())
- case_blocks.append(case_block)
+ case_blocks.append(case_block.as_text())
```

If you look at the implementation of `case_block.as_text()` you'll see that it already does the decoding and is equivalent to `ElementTree.tostring(case_block.as_xml()).decode('utf-8')`

Comment (mkangia) [reply corehq/apps/hqcase/utils.py]: thank you @snopoke 
Updated, https://github.com/dimagi/commcare-hq/pull/25624/commits/7c8d5f5b47fbaa676dc5bcaac087de5ded344661

**Comment by mkangia** [comment]
@snopoke this is ready for merge. Can you please review the changes post your last review and merge it if its good?
Thank you.

**Review by snopoke — APPROVED** [review]

**Pull request merged**